### PR TITLE
Deactivate public in dev/stage environment

### PR DESCRIPTION
**Pull request by wachterjohannes**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | no
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | https://github.com/sulu/sulu/pull/3015
| License | MIT
| Documentation PR | none

See https://github.com/sulu/sulu/pull/3015

### CODE REVIEW

File: app/config/admin/config_prod.yml [hunk 1]
```diff
@@ -23,3 +23,11 @@ monolog:
...
         console:
             type:  console
+
+sulu_http_cache:
+    handlers:
+        public:
+            max_age: 240
+            shared_max_age: 240
+            use_page_ttl: true
+            enabled: true
```
Comment (danrot) [inline app/config/admin/config_prod.yml]: Isn't that unnecessary? All of this are default values, also the `enabled` flag, if the application runs in the prod environment. Same for the other `config_prod` file.

**Comment by alexander-schranz** [comment]
the stage should be as near as possible to the prod environment. would this not deactivate the cache on stage environment? its hard when you test on stage a whole system and bugs with caching only appears on prod.

**Comment by danrot** [comment]
@alexander-schranz Actually that's also what I thought... See my other comment in sulu/sulu#3015. I would say we check there for `dev` and `test`, as it is also done in the kernel for the bundles. /cc @wachterjohannes

**Comment by wachterjohannes** [comment]
@danrot @alexander-schranz fixed

**Comment by danrot** [comment]
Thank you, but please create the branch on your fork the next time 😉

**Pull request merged**